### DISCUSSION TITLE
Creating an issue on Github Android app with comment over 65k characters will fail with no error output in the app, but it does on the website.

### DISCUSSION THREAD

**Discussion by yuukiAme** [community/community#121927] [Mobile] [answered]
When I tried to create an issue with a repo to repor an issue, it did not post on the issue page.

I tried again on the website and it clearly told me that I had too many characters but when I did it in the Github Android app, the app did not report on any errors which made me believe the issue was reported.

**Comment by kangarruu** [comment, marked as answer]
@yuukiAme Thanks for reporting that! I opened up an internal issue to track this so we could investigate and provide a better experience.

**Reply by mannyg253** [reply]
Dead people are hot
